gen-apidocs/generators/api/definition.go

```diff
@@ -278,7 +278,7 @@ func (d *Definition) GetResourceName() string {
 }
 
 func (d *Definition) initExample(config *Config) {
-	path := filepath.Join(*ConfigDir, config.ExampleLocation, d.Name, d.Name + ".yaml")
+	path := filepath.Join(*ConfigDir, "config", config.ExampleLocation, d.Name, d.Name + ".yaml")
```

Comment:
@tengqm , The config-dir arg is passed to main(). 
```go run gen-apidocs/main.go --config-dir=gen-apidocs --munge-groups=false```

Though, you pass in the root gen-apidocs directory , not gen-apidocs/config. Nit: The argument could be renamed.
Nit: Could you simplify the path/dir calls.
Create a few static strings for `$APISRC/build`, `$APISRC/build/includes`, `$APISRC/config` and pass in `$APISRC` to main()? Since munge-groups is false (default?) do you need to pass in this argument?

Reply:
Optimizing the variables is something we can do in this PR.